Lessons/Additional/AdditionalTasks.cs

```diff
@@ -0,0 +1,132 @@
+﻿namespace Lessons.Additional
+{
+    internal class AdditionalTasks
+    {
+        public static void Task1_FindSecondMaximumNumber()
+        {
```

Comment:
here you iterate to array twice, 
1. find max
2. find second max

and you doing O(2n)

lets optimise it and solve task using only 1 for cycle and achieve O(n)